### DISCUSSION TITLE
boost 1.78.0 --tapped

### DISCUSSION THREAD

**Discussion by DanielKui** [xmake-io/xmake-repo#2370] [Q&A]
请教一下，add_requires("boost 1.78.0",  { configs = {shared = true, system = true, locale = true}})
如何去掉编译参数--layout=tagged-1.66呢？有这个编译参数会导致-mt后缀在linux下

![image](https://github.com/xmake-io/xmake-repo/assets/578118/67358c95-75c3-4423-a24c-6f784ee80a67)
![image](https://github.com/xmake-io/xmake-repo/assets/578118/ad9fa74b-e030-4ebf-835c-cb7369a5e33e)
![image](https://github.com/xmake-io/xmake-repo/assets/578118/88ac67f6-baea-4b9e-a728-103e419075b0)

**Comment by waruqi** [comment]
这个加了又不影响啥，去来干嘛。。

**Comment by DanielKui** [comment]
感谢回复，对了，大佬  ,protobuf-cpp最好有个config 配置支持protobuf还是protobuf-lite

![image](https://github.com/xmake-io/xmake-repo/assets/578118/7ecd66f7-5814-444a-afcd-d62291dab070)

**Reply by waruqi** [reply]
> 感谢回复，对了，大佬 ,protobuf-cpp最好有个config 配置支持protobuf还是protobuf-lite

那是 protobuf 自己生成的 filename 变了，所以得同步下，否则 link error

你这没事去来干嘛，不要瞎折腾。
